configuration/dto.go

```diff
@@ -35,6 +35,7 @@ type RealmConfiguration struct {
 	RedirectCancelledRegistrationURL  *string   `json:"redirect_cancelled_registration_url,omitempty"`
 	RedirectSuccessfulRegistrationURL *string   `json:"redirect_successful_registration_url,omitempty"`
 	OnboardingRedirectURI             *string   `json:"onboarding_redirect_uri,omitempty"`
+	IdentificationURI                 *string   `json:"identification_uri,omitempty"`
```

Comment:
I do not understand why this is needed ?

Reply:
You're right, my bad! I think I used it in a past version of my implementation in keycloak-bridge, thanks a lot for pointing this out